lib/resources/r4/questionnaire_response.yaml

```diff
@@ -23,4 +23,168 @@ fields:
     example: |
       {
         "id": "SH-12508041"
-      }
\ No newline at end of file
+      }
+
+  - name: questionnaire
+    required: 'Yes'
+    type: Questionnaire
+    url: http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.questionnaire
+    description: The Questionnaire that defines and organizes the questions for which answers are being provided.
+    action: update
+    example: |
+      {
+        "questionnaire": "https://fhir-ehr-code.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Questionnaire/SH-12508041"
+      }
+
+  - name: status
+    required: 'Yes'
+    type: code
+    url: http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.status
+    description: The position of the questionnaire response within its overall lifecycle.
+    action: update
+    example: |
+      {
+        "status": "in-progress"
+      }
+  
+  - name: subject
+    required: 'Yes'
+    type: Reference
+    url: http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.subject
+    description: The subject of the questionnaire response. This could be a patient, organization, practitioner, device, etc. This is who/what the answers apply to, but is not necessarily the source of information.
+    action: update
+    example: |
+      {
+        "subject": {
+          "reference": "Patient/12508041"
+        }
+      }
+
+  - name: encounter
+    required: 'No'
+    type: Reference
+    url: http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.encounter
+    description: The Encounter during which this questionnaire response was created or to which the creation of this record is tightly associated.
+    action: update
+    example: |
+      {
+        "encounter": {
+          "reference": "Encounter/36209678"
+        }
+      }
+
+  - name: authored
```

Comment:
authored, we dont actually write... should it still be listed here? we returned it still but ignorre it on an update

Reply:
let's get rid of it. It's not a supported field.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/669/commits/6b1e581f4f01eb2d8a1a69a8a0f857da78c0fcc3